triliovault/values_overrides/tls.yaml

```diff
@@ -0,0 +1,63 @@
+---
+pod:
+  security_context:
+    cinder_api:
```

Comment:
We don't need cinder_api, right?

Reply:
Yes, we don't need that.
Changes are in progress.